src/components/CommunicationAndPartners/index.tsx

```diff
@@ -136,9 +134,9 @@ function CommunicationAndPartners(props: CommunicationAndPartnersProps) {
             page: contactPage,
             pageSize: contactPageSize,
             name: contactSearch,
-            country,
+            country: defaultCountryOption ? defaultCountryOption[0].id : undefined,
```

Comment:
```suggestion
            country: defaultCountryOption?.[0].id,
```

But, it's better to accept a single country option if model support single country
and accept list of country as options if model supports multiple countries.